SnipsPlatformTests/SnipsPlatformTests.swift

```diff
@@ -115,7 +117,7 @@ class SnipsPlatformTests: XCTestCase {
         }
         
         try! self.snips?.startSession(canBeEnqueued: false, sendIntentNotRecognized: true)
-        wait(for: [onIntentNotRecognizedExpectation], timeout: 20)
+        wait(for: [onIntentNotRecognizedExpectation], timeout: 40)
```

Comment:
wait... what?

Reply:
Due to the ASR partial text setting, the platform runs more slowly. I had to increase this timeout